HGCalAnalysis/plugins/HGCalAnalysis.cc

```diff
@@ -836,7 +921,75 @@ HGCalAnalysis::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
 		multiclus_slopeY.push_back(multiClusters[i].y());
 		multiclus_cluster2d.push_back(cl2dIndices);
 		multiclus_cl2dSeed.push_back(cl2dSeed);
+		multiclus_firstLay.push_back(*layers.begin());
+		multiclus_lastLay.push_back(*layers.rbegin());
+		multiclus_NLay.push_back(layers.size());
+		if(multiClusters[i].size()<3)
+		  	{
+			multiclus_pcaAxisX.push_back(-2.);
```

Comment:
I also think keeping this value makes sense. However, I guess the PCA will mostly be used by Florian and Arthur (maybe Ed?), so what about just adding an *additional* configurable flag (default false) to either stores these variables in the ntuple or not to avoid blowing up the ntuple size unnecessarily?